net/device/IotHub.cs

```diff
@@ -123,6 +123,7 @@ public async Task RunAsync(CancellationToken ct)
 
                         ++_totalMessagesSent;
                         _logger.Metric("TotalMessagesSent", _totalMessagesSent);
+                        _logger.Metric("MessageDelayMinutes", (DateTime.UtcNow - pendingMessage.CreationTimeUtc).TotalMinutes);
```

Comment:
I haven't synced with the spec yet, but I added this for my interim runs so I can see how long messages are delayed.